src/styles/_stars.scss

```diff
@@ -0,0 +1,33 @@
+@use './mixins' as *;
+@use './variables' as *;
+@use './card' as *;
+
+.stars {
+  @include center;
+
+  line-height: 14px;
+  margin-top: 16px;
+  width: $primary-width;
+
+  &__star {
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    background-image: url('../images/star-active.svg');
+    background-repeat: no-repeat;
+  }
+
+  &__star:nth-child(n + 5) {
+    background-image: url('../images/star.svg');
+    margin-right: 33px;
```

Comment:
you should use space-between to make space between all stars elements and stars__reviews element so it would work if card change it's width